### PR TITLE
Add Aruba IAPs

### PR DESCRIPTION
**Pull request by moetezz**
This PR contributes more models to HPE Aruba access points such as 500H Series, 570 Series, 630 Series.

### CODE REVIEW

File: device-types/HPE/Aruba-IAP-505H.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+console-ports:
+  - name: USB 2.0
+    type: usb-a
+    label: USBA
+power-ports:
+  - name: DC Power Interface
+    type: dc-terminal
+    label: DC Power
+interfaces:
+  - name: BLUETOOTH / Zigbee
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-505H.yaml:21]: Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-505H.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    description: Auto-sensing link speed (10/100/1000BASE-T)
+  - name: Ethernet 4
+    type: 1000base-t
+    label: E4
+    description: Auto-sensing link speed (10/100/1000BASE-T)
+  - name: WIFI / 2.4 GHz Radio
+    type: ieee802.11ax
+    label: Radio 0
+    description: Integrated
+  - name: WIFI / 5GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-505H.yaml:55]: Is the interface name actually `WIFI / 5GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-574.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+airflow: passive
+comments: '[Aruba 570 Series Datasheet](https://www.arubanetworks.com/assets/ds/DS_AP570Series.pdf)'
+weight: 2.7
+weight_unit: kg
+console-ports:
+  - name: Console Interface
+    type: usb-c
+    label: USBC
+interfaces:
+  - name: BLUETOOTH / Zigbee
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-574.yaml:17]: Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-574.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    type: 1000base-t
+    label: E1
+    description: Auto-sensing link speed
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI/2.4 GHz Radio
+    type: ieee802.11ax
+    label: Radio 1
+    description: Integrated
+  - name: WIFI/5GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-574.yaml:37]: Is the interface name actually `WIFI/5GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-574.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    description: Auto-sensing link speed (100/1000/2500BASE-T)
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: Ethernet 1
+    type: 1000base-t
+    label: E1
+    description: Auto-sensing link speed
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI/2.4 GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-574.yaml:33]: Is the interface name actually `WIFI/2.4 GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-575.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+airflow: passive
+comments: '[Aruba 570 Series Datasheet](https://www.arubanetworks.com/assets/ds/DS_AP570Series.pdf)'
+weight: 2.5
+weight_unit: kg
+console-ports:
+  - name: Console Interface
+    type: usb-c
+    label: USBC
+interfaces:
+  - name: BLUETOOTH / Zigbee
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-575.yaml:17]: Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-575.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    type: 1000base-t
+    label: E1
+    description: Auto-sensing link speed
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI/2.4 GHz Radio
+    type: ieee802.11ax
+    label: Radio 1
+    description: Integrated
+  - name: WIFI/5GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-575.yaml:37]: Is the interface name actually `WIFI/5GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-575.yaml [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    description: Auto-sensing link speed (100/1000/2500BASE-T)
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: Ethernet 1
+    type: 1000base-t
+    label: E1
+    description: Auto-sensing link speed
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI/2.4 GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-575.yaml:33]: Is the interface name actually `WIFI/2.4 GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-577.yaml [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+airflow: passive
+comments: '[Aruba 570 Series Datasheet](https://www.arubanetworks.com/assets/ds/DS_AP570Series.pdf)'
+weight: 2.1
+weight_unit: kg
+console-ports:
+  - name: Console Interface
+    type: usb-c
+    label: USBC
+interfaces:
+  - name: BLUETOOTH/Zigbee
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-577.yaml:17]: Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-635.yaml [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+  - name: USB 2.0
+    type: usb-a
+    label: USBA
+power-ports:
+  - name: 12Vdc
+    type: dc-terminal
+    maximum_draw: 26
+    label: DC
+interfaces:
+  - name: BLUETOOTH / Zigbee
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-635.yaml:22]: Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-635.yaml [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+    poe_type: type4-ieee802.3bt
+  - name: WIFI / 2.4 GHz Radio
+    type: ieee802.11ax
+    label: Radio 0
+    description: Integrated
+  - name: WIFI / 5GHz Radio
+    type: ieee802.11ax
+    label: Radio 1
+    description: Integrated
+  - name: WIFI / 6GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-635.yaml:46]: Is the interface name actually `WIFI / 6GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-635.yaml [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    type: 2.5gbase-t
+    label: E1
+    description: Auto-sensing link speed (100/1000/2500BASE-T)
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI / 2.4 GHz Radio
+    type: ieee802.11ax
+    label: Radio 0
+    description: Integrated
+  - name: WIFI / 5GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-635.yaml:42]: Is the interface name actually `WIFI / 5GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-577.yaml [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+    type: 1000base-t
+    label: E1
+    description: Auto-sensing link speed
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI/2.4 GHz Radio
+    type: ieee802.11ax
+    label: Radio 1
+    description: Integrated
+  - name: WIFI/5GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-577.yaml:37]: Is the interface name actually `WIFI / 5GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-635.yaml [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+    description: Auto-sensing link speed (100/1000/2500BASE-T)
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: Ethernet 1
+    type: 2.5gbase-t
+    label: E1
+    description: Auto-sensing link speed (100/1000/2500BASE-T)
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI / 2.4 GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-635.yaml:38]: Is the interface name actually `WIFI / 2.4 GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

File: device-types/HPE/Aruba-IAP-577.yaml [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+    description: Auto-sensing link speed (100/1000/2500BASE-T)
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: Ethernet 1
+    type: 1000base-t
+    label: E1
+    description: Auto-sensing link speed
+    poe_mode: pd
+    poe_type: type4-ieee802.3bt
+  - name: WIFI/2.4 GHz Radio
```
Comment (danner26) [inline device-types/HPE/Aruba-IAP-577.yaml:33]: Is the interface name actually `WIFI / 2.4 GHz Radio`? The name here should match what you see on the CLI or programmatic interface if there is one

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by moetezz** [comment]
For all the mentioned examples. I didn't use a CLI, but that is what I inferred from the device specs. I found that these devices have multiple radios and antennas for each Frequency/wireless technology

File: device-types/HPE/Aruba-IAP-505H.yaml (hunk 2 above)
Comment (moetezz) [reply device-types/HPE/Aruba-IAP-505H.yaml]: I made the change to best reflect what's on the DataSheet

File: device-types/HPE/Aruba-IAP-505H.yaml (hunk 1 above)
Comment (moetezz) [reply device-types/HPE/Aruba-IAP-505H.yaml]: I changed it according to the Data Sheet.

File: device-types/HPE/Aruba-IAP-574.yaml (hunk 3 above)
Comment (moetezz) [reply device-types/HPE/Aruba-IAP-574.yaml]: _I made the change to best reflect what's on the Aruba DataSheet_

**Comment by danner26** [comment]
Yeah, I would update these to have the proper inteface names and then if you want to use the label as a way to specify wifi that is fine. The key details if that the interface name matches what the device has

**Pull request closed without merging**